apps/full-site-editing/full-site-editing-plugin/premium-content/premium-content.php

```diff
@@ -0,0 +1,278 @@
+<?php declare( strict_types = 1 );
+/**
+ * Plugin Name:     Premium Content
+ * Description:     Example block written with ESNext standard and JSX support – build step required.
+ * Version:         0.1.0
+ * Author:          The WordPress Contributors
+ * License:         GPL-2.0-or-later
+ * Text Domain:     create-block
+ *
+ * @package create-block
+ */
+
+namespace A8C\FSE\Earn\PremiumContent;
+
+use RuntimeException;
+use function register_block_type;
+use function plugin_dir_url;
+use function apply_filters;
+use A8C\FSE\Earn\PremiumContent\Premium_Content_Dom;
+use A8C\FSE\Earn\PremiumContent\SubscriptionService\{
+	Subscription_Service,
+	Jetpack_Token_Subscription_Service,
+	WPCOM_Token_Subscription_Service
+};
+
+const PAYWALL_FILTER = 'earn_premium_content_subscription_service';
+
+require_once __DIR__ . '/subscription-service/include.php';
+require_once __DIR__ . '/premium-content-dom.php';
+
+/**
+ * Registers all block assets so that they can be enqueued through the block editor
+ * in the corresponding context.
+ *
+ * @see    https://developer.wordpress.org/block-editor/tutorials/block-tutorial/applying-styles-with-stylesheets/
+ * @return void
+ */
+function premium_content_block_init() {
```

Comment:
https://code.a8c.com/D42446 is dying here because we are now double-declaring the code here.
We will need a few checks for double declaration until we get this transition sorted.
```
[27-Apr-2020 11:27:33 UTC] PHP Fatal error:  Cannot redeclare A8C\FSE\Earn\PremiumContent\premium_content_block_init() (previously declared in /home/wpcom/public_html/wp-content/mu-plugins/premium-content/premium-content.php:36) in /home/wpcom/public_html/wp-content/plugins/full-site-editing-plugin/88442264/premium-content/premium-content.php on line 95
```